src/components/Professor/ProfessorCard/style.scss

```
@@ -0,0 +1,51 @@
+@use '@/style/package' as *;
+
+.professor {
+  display: grid;
+  grid-row-gap: 10px;
```

```suggestion
  gap: 1rem 0;
```
grid-row-gap, grid-column-gap 의 shortcut인 gap을 사용해보자

---

완!